packages/amplication-client/src/style/dark-css-variables.css

```diff
@@ -41,6 +41,7 @@
   --image-my-resources: url("../assets/images/mono/my-resources-dark.svg");
   --image-generating: url("../assets/images/mono/generating-dark.svg");
   --image-create-service-wizard : url("../assets/images/mono/service-wizard-create.svg");
+  --plugin-installation-empty: url("../assets/images/mono/plugin-installation-empty.svg");
```

Comment:
to keep the current convention and until we completely remove the light team support form the code, please call the file with the suffix "-dark"